src/components/ui/check-chip-group.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import itemAmount from 'ui/utils/item-amount.js';
+
+  export let classname = '';
+  export let chipwrapperclass = '';
+  export let chipclass = '';
+  export let inputclass = '';
+  export let small = false;
+  export let outline = false;
+
+  export let values;
+  export let labels = null;
+  export let name;
+  export let maxChecked = null;
+  $: currentChecked = values.reduce((acc, elt) => acc + elt.checked, 0);
+  const titleObj = { title: `Can only select ${itemAmount(maxChecked, 'value')}.` };
+
+  if (values.length === 0) {
+    console.error('Must have at least one value in the checkbox group.');
+  }
+
+  if (labels !== null && values.length !== labels.length) {
+    console.error('Must have as many labels as there is values.');
+  }
+
+  const dispatch = createEventDispatcher();
+  function checkLimit(evt) {
+    if (maxChecked !== null && currentChecked + evt.target.checked  > maxChecked) {
```

Comment:
I would prefer separating the condition for `evt.target.checked` so that if for whatever reason or error the currentChecked passed the max and this was unchecking it, it should work.
```suggestion
    if (maxChecked !== null && currentChecked >= maxChecked && evt.target.checked) {
```

Reply:
I don't know why, but when I tried to do it, it didn't work. It didn't work when I added `!evt.target.checked`, it didn't work when I added `!!evt.target.checked`, and this worked.
I don't know what is going on and at this point I'm not in the mood to bother investigating. This works